spaceship/README.md

```diff
@@ -133,6 +133,15 @@ export FASTLANE_SESSION='---\n- !ruby/object:HTTP::Cookie\n  name: DES5c148586df
 
 Copy everything from `---\n` to your CI server and provide it as environment variable named `FASTLANE_SESSION`.
 
+#### Bypass device and use SMS for verification
+
+If you have a trusted device configured for your Apple account, then Apple will not send a SMS code to your phone when you try to generate a web session with fastlane — instead a code will be displayed on one of your account's trusted devices. This can be problematic if you are trying to authenticate, but don't have access to a device. To circumvent the device and use SMS instead, take the following steps:
```

Comment:
Can we reword this to the following?
```md
Apple will not send a SMS code to your phone if you have a trusted device configured for your Apple account when you try to generate a web session with fastlane — instead a code will be displayed on one of your account's trusted devices. This can be problematic if you are trying to authenticate but don't have access to a trusted device. Take the following steps to circumvent the device and use SMS instead:
```

Reply:
Can you also add underscores around "fastlane" in here too? `_fastlane_`?